hack/mirror/walrus-images.txt

```diff
@@ -11,4 +11,4 @@ docker.io/sealio/casdoor:v1.344.0-seal.1
 docker.io/sealio/terraform-deployer:v1.5.7-seal.1
 docker.io/sealio/opentofu-deployer:v1.6.0-seal.1
 docker.io/sealio/hermitcrab:v0.1.3
-docker.io/sealio/walrus:v0.4.0
+docker.io/sealio/walrus:v0.5.0
```

Comment:
When we release v0.5.1, it becomes out of date again. Can we handle it in the CI script?